files/build_templates/docker_image_ctl.j2

```diff
@@ -170,16 +170,27 @@ function postStartAction()
            ip netns exec "$NET_NS" ip addr add 127.0.0.1/16 dev lo
            ip netns exec "$NET_NS" ip addr del 127.0.0.1/8 dev lo
 
+           slot_id=$(python3 -c 'import sonic_platform.platform; platform_chassis = sonic_platform.platform.Platform().get_chassis(); print(platform_chassis.get_my_slot())' 2>/dev/null)
+           supervisor_slot_id=$(python3 -c 'import sonic_platform.platform; platform_chassis = sonic_platform.platform.Platform().get_chassis(); print(platform_chassis.get_supervisor_slot())' 2>/dev/null)
+
+           # Create eth1 in database instance
+	   if [[ "${slot_id}" == "${supervisor_slot_id}" ]]; then
+		   ip link add name ns-eth1"$NET_NS" type veth peer name eth1@"$NET_NS"
+		   ip link set dev eth1@"$NET_NS" master br1
+		   ip link set dev eth1@"$NET_NS" up
+	   else
+		   ip link add name ns-eth1"$NET_NS" link eth1-midplane type macvlan mode bridge
+	   fi
+
            # Create eth1 in database instance
            ip link add name ns-eth1"$NET_NS" link eth1-midplane type macvlan mode bridge
```

Comment:
Do we need this line, seems same as line 182 ?

Reply:
Right, we don't need this line.

Reply:
Should be fixed now.